whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageDequeue.java

```diff
@@ -67,6 +67,9 @@ public class MessageDequeue implements ApplicationService {
     @Value("${mail.dequeue.interval:1000}")
     private int intervalMs;
 
+    @Value("${mailupdates.passwd.error:false}")
+    private boolean passwdError;
```

Comment:
mailupdates.use.password.allowed?  By default should be true 
and variable name is passwordAllowed

Reply:
no, by default is not allowed
` If the flag is OFF then return a warning (make this the default)`

Reply:
I rename it by `errorIfPassword`